deploy/crds/operators_v1_catalogsourceconfig_crd.yaml

```diff
@@ -44,3 +44,12 @@ spec:
             packages:
               type: string
               description: Comma separated list of operator(s) without spaces which will be enabled in the target namespace
+            csDisplayName:
+              type: string
+              description: DisplayName is passed along to the CatalogSource to be used as a pretty name.
+            csPublisher:
+              type: string
+              description: Publisher is passed along to the CatalogSource to be used to define what entity published the operator manifest(s) from the OperatorSource.
```

Comment:
`to the OperatorSource`

Reply:
I don't think `to the OperatorSource` makes sense here. `operator manifest(s) associated with the OperatorSource` is a bit clearer than `from`.

If I hadn't copied the comment from the `go` type file, I would have gone with something simple like - `It represents the entity that published the the operator(s) specified in packages`.

Reply:
Well "from the OperatorSource" didn't make sense to me either. I too prefer your latter comment. We can clarify the comment in the go type file in a follow up PR. No reason to propagate something that doesn't sound right in more places.